v3/newrelic/collector.go

```diff
@@ -231,6 +232,26 @@ func collectorRequest(cmd rpmCmd, cs rpmControls) rpmResponse {
 	return resp
 }
 
+func removeLicenseFromURL(u string) string {
+	rawURL, err := url.Parse(u)
+	if err != nil {
+		return ""
+	}
+
+	query := rawURL.Query()
+	licenseKey := query.Get("license_key")
+
+	// License key length has already been checked, but doing another
+	// conservative check here.
+	if n := len(licenseKey); n > 4 {
+		query.Set("license_key", string(licenseKey[0:2]+".."+licenseKey[n-2:]))
+	} else {
+		query.Set("license_key", licenseKey)
```

Comment:
this line (else) is redundant, license key is already set.

Reply:
Of course, thanks!